docs/reference/cat/recovery.asciidoc

```diff
@@ -1,16 +1,64 @@
 [[cat-recovery]]
 === cat recovery
 
-The `recovery` command is a view of index shard recoveries, both on-going and previously
-completed. It is a more compact view of the JSON <<indices-recovery,recovery>> API.
+Returns information about ongoing and completed index shard recoveries, similar
+to the <<indices-recovery, indices recovery>> API.
 
-A recovery event occurs anytime an index shard moves to a different node in the cluster.
-This can happen during a snapshot recovery, a change in replication level, node failure, or
-on node startup. This last type is called a local store recovery and is the normal
-way for shards to be loaded from disk when a node starts up.
 
-As an example, here is what the recovery state of a cluster may look like when there
-are no shards in transit from one node to another:
+[[cat-recovery-api-request]]
+==== {api-request-title}
+
+`GET /_cat/recovery/{index}`
+
+
+[[cat-recovery-api-desc]]
+==== {api-description-title}
+
+The cat recovery API returns information about index shard recoveries, both
+ongoing and completed. It is a more compact view of the JSON
+<<indices-recovery,indices recovery>> API.
+
+A recovery event occurs anytime an index shard moves to a different node in the
+cluster. This can happen during a snapshot recovery, a change in replication
+level, node failure, or on node startup. This last type is called a local store
+recovery and is the normal way for shards to be loaded from disk when a node
+starts up.
+
+As an example, here is what the recovery state of a cluster may look like when
+there are no shards in transit from one node to another:
+
+
```

Comment:
Is there an example missing here?

(I also twitch over "may" vs "might" ... I would reflexively use might because "may implies permission". But that's one of those editorial stickles where the distinction doesn't really matter to most readers.)

Reply:
Thanks for catching this @debadair. I intended to remove this sentence entirely. Removed with c7f88d63d9f.

The example has been moved to https://github.com/elastic/elasticsearch/pull/45345/files#diff-0e907dfad62124d1dc58267b446d2c51R61